LUSDChickenBonds/src/NFTArtwork/ChickenInTraitWeights.sol

```diff
@@ -0,0 +1,130 @@
+pragma solidity ^0.8.11;
+
+import "./EggTraitWeights.sol";
+
+
+contract ChickenInTraitWeights is EggTraitWeights {
+    uint256 constant EGG_CHICKEN_REPEAT_FACTOR = 120e16;
+    // See: https://www.desmos.com/calculator/fctburkmtg
+    uint256 constant R3 = 52e16;
+    uint256 constant ALPHA3 = 1e18 / 3;
```

Comment:
3 should be explicitly converted to `uint256`

Reply:
Sorry, it wasn’t even compiling! :sweat_smile: